css/styleEx.css

```diff
@@ -0,0 +1,327 @@
+
+.contact p {
+    margin: 15px 0;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+}
+
+@media only screen and (max-width: 800px) {
+    header {
+        width: 100%;
+        flex-wrap: nowrap;
+        display: flex; 
+        flex-direction: column-reverse;
+        /* justify-content: center; */
+        /* width: 100%; */
+        align-items: center;
+    }
+    /* header nav a {
+        width: 100%;
+        /* display: flex;
+        justify-content: center; 
+        align-content: center;*/
+    
+    header .logo {
+        width: 25%;        
+        display: flex;         
+        justify-content: center; 
+        align-items: center;
+                    
+    }
+    .cta .cta-text {
+        width: 100%;
+        /* align-content: center; */
+        display: flex;         
+        justify-content: center; 
+        align-items: center;
+
+    }
+    .cta img {
+        display: none;
+    }
+    .top-content .text-content {
+        width: 100%;
+        /* display: flex;        
+        justify-content: center; */
+    } 
+    .middle-img {
+        width: 100%;
+        /* display: flex;        
+        justify-content: center; */
+    } 
+    .bottom-content {
+        width: 100%;
+        /* display: flex;        
+        justify-content: center; */
+    }
+    /* section .contact {
+        margin-left: -200px;        
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        align-items: flex-start;
+        align-content: left;       
+        align-self: flex-start;
+       
+    }       */
+}
+@media only screen and (max-width: 500px) {
+    header {
+        width: 100%;
+        flex-wrap: nowrap;
+        display: flex; 
+        flex-direction: column-reverse;
+        /* justify-content: center; */
+        /* width: 100%; */
+        align-items: center;
+        border: 0.5px solid gray;
+    }
+    header nav {
+        width: 100%;        
+        display: flex;
+        flex-wrap: wrap;
+        flex-direction: column;
+        justify-content: center; 
+        align-content: center;
+        /* border: 1px solid black */
+    }
+    header nav a {
+        width: 100%;
+        font-size: 24px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center; 
+        align-content: center;
+        border: 0.5px solid gray;
+        text-align: center;
+        padding: 10px;
+    }
+    
+    header .logo {
+        width: 50%;        
+        display: flex;         
+        justify-content: center; 
+        align-items: center;       
+                    
+    }
+    .cta {
+        display: flex;
+        justify-content: center; 
+        align-items: center;  
+        
+    }
+    .cta .cta-text {
+        width: 100%;
+        /* align-content: center; */
+        display: flex;         
+        justify-content: center; 
+        align-items: center;
+
```

Comment:
It looks like you had the same issue on mobile that you did on tablet. Adjusting some `margin-left` would have been a  good solution.